packages/clerk-js/src/ui/elements/CodeControl.tsx

```
@@ -160,15 +161,21 @@ export const OTPResendButton = () => {
 export const OTPCodeControl = React.forwardRef<{ reset: any }>((_, ref) => {
   const [disabled, setDisabled] = React.useState(false);
   const refs = React.useRef<Array<HTMLInputElement | null>>([]);
+  const hiddenInputRef = React.useRef<HTMLInputElement>(null);
   const firstClickRef = React.useRef(false);
 
-  const { otpControl, isLoading, isDisabled, centerAlign = true } = useOTPInputContext();
+  const { otpControl, isLoading, isDisabled, centerAlign = true, passwordManagerOffset = 40 } = useOTPInputContext();
```

ESLint

[error] 'passwordManagerOffset' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)